src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -16,6 +16,7 @@
     <SupportsX86Intrinsics Condition="'$(Platform)' == 'x64' or ('$(Platform)' == 'x86' and '$(TargetsUnix)' != 'true')">true</SupportsX86Intrinsics>
     <ILLinkSharedDirectory>$(MSBuildThisFileDirectory)ILLink\</ILLinkSharedDirectory>
     <Is64Bit Condition="'$(Platform)' == 'arm64' or '$(Platform)' == 'x64'">true</Is64Bit>
+    <UseMinimalGlobalizationData Condition="'$(IsiOSLike)' == 'true' OR '$(TargetsBrowser)' == 'true'">true</UseMinimalGlobalizationData>
```

Comment:
I think we should exclude catalyst here since it uses the system ICU so doesn't run into this issue.